### ISSUE TITLE
UG hard to follow

### ISSUE THREAD

**Issue by huien77** [huien77/pe#9]
![image.png](https://raw.githubusercontent.com/huien77/pe/main/files/71993ed1-377a-4d19-ab9d-7b0d648a250e.png)

for this part of the UG, I felt it might be better if the command comes before the screenshot instead of being all before the screenshots, I had to scroll up and down to see the command and then match it to the screenshot

**Comment by nus-se-script** [comment]
# Team's Response
Our intention is to provide users with a short example of what they can input. If the user is still confused, they would be able to refer to additional information below with both the input AND the output. Thus this should not hinder understanding.

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

-------------------

## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.Medium`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

-------------------
